cmp/grid/impl/GridPersistenceModel.js

```diff
@@ -189,14 +161,26 @@ export class GridStateModel {
         return ret;
     }
 
-    @debounced(500)
-    saveStateChange() {
-        const {state} = this;
+    @action
+    patchState(updates) {
+        this.state = {...this.state, ...updates};
+    }
+
```

Comment:
Should we consider a straightforward `setState()` method, which simply replaces the entire state blob?